content/ai/concepts/neural-networks/terms/convolutional-neural-networks/convolutional-neural-networks.md

```diff
@@ -0,0 +1,33 @@
+---
+Title: 'Convolutional Neural Networks'
+Description: 'Convolutional Neural Networks are a type of neural network that are primarily used for image classification.'
+Subjects:
+  - 'Computer Science'
+Subjects:
+  - 'Deep Learning'
+  - 'Computer Science'
+Tags:
+  - 'AI'
+  - 'Neural Networks'
+  - 'CNN'
+CatalogContent:
+  - 'machine-learning'
+  - 'paths/machine-learning-engineer'
+---
+
+# What is a CNN?
+
+CNNs are a **type of neural network** that are primarily used for image classification. They are also used for other tasks such as object detection, image segmentation, and natural language processing.
+
+CNNs are inspired by the **visual cortex** of the brain. The visual cortex is the part of the brain that processes visual information. 
+
+# How does a CNN work?
```

Comment:
```suggestion
## How does a CNN work?
```
We don't use h1 headers. That's reserved for the title provided by the metadata.